### ISSUE TITLE
508-defect-2 [COGNITION]: VAOS Express Care - Bold text on headings must be consistently marked up as headings

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#19301]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I would like to navigate through my appointment cards by heading and hear key information like appointment status, location, and phone numbers, read aloud.

## Details

The VA 508 office flagged some of our bold heading-like text in their quick test. We have some bold text coded as H4s, and others as strong or div tags. We discussed this in the monthly call on Wednesday 1/28, and agreed these should be H4s consistently.

From VA's email to me:
> I was browsing around by heading and found another area that looks like a heading but isn’t coded as a heading.  The “How to joint your video appointment” is Labeled as Heading 4 yes “Main phone” or “Confirmed” is not labeled as a heading.  I’m not necessarily saying they should be headings however our office generally has a rule of thumb if they look like a heading visually (bold bigger font) they should be labeled as a heading.  What do you all think?

## Acceptance Criteria

- [ ] All highlighted text blocks are coded as H4s
- [ ] No axe-core violations appear
- [ ] Design has had a chance to spot check the changes and give a thumbs up

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2021-01-29 at 4 29 30 PM](https://user-images.githubusercontent.com/934879/106334952-6289c080-6251-11eb-859a-c013e3b29fa1.png)

---
![Screen Shot 2021-01-29 at 4 30 10 PM](https://user-images.githubusercontent.com/934879/106334970-6a496500-6251-11eb-8e01-a25bb767bc3c.png)

**Comment by outerpress** [comment]
@1Copenut just to clarify, we are just talking about coding these as H4, but can still visually style them as bold source sans text?

I did a couple of tests with these appointment cards using regular H4's and they created a really uneven visual hierarchy that was difficult to scan. The headings added too much noise to the page because they weren't breaking up paragraphs of content - there was just a little content in between each. Adding more space between each section helped a little with that issue, but made it worse by spreading the content across the page too much.

**Comment by jbalboni** [comment]
@1Copenut Some of that text is marked up inside definition lists, I assume we should remove that and just use headings?

**Comment by 1Copenut** [comment]
@outerpress Correct. We're suggesting to code these as H4s, but maintain the current visual styling of bold, sans serif text.

@jbalboni Yes, let's remove the definition lists and code them as headings. We had a long discussion with the VA 508 office about the merits of lists vs. headings. One of VA's SMEs said headings are the preferred means to traverse content quickly.
